### ISSUE TITLE
Add accessibility

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/MapML#1]

**Comment by Malvoz** [comment]
Skimming through the spec I didn't find anything describing how maps are to be navigated using keyboard only, such as basic navigation or any eventual shortcuts. 

Although using the up/right/down/left arrows makes sense and seems to be the implied method, what happens is if the map has a parent element with an overflow (usually the case with `<body>` on the Y-axis) and you press the up/down arrow, the parent element will scroll simultaneously until the map is not visible. This is obviously an accessibility concern. 

I noted holding `shift` while pressing the arrow keys prevents the overflowing parent from scrolling along, which is the desired effect.

<hr>

Also it might be wise to consider how maps works with [spatial navigation](https://github.com/WICG/spatial-navigation) enabled by UAs.

**Comment by prushforth** [comment]
Accessibility is obviously important, and so far I've mostly relied on what Leaflet has provided. It definitely needs to be discussed in the spec, and I hope to get others involved to help with this. So, your comment is well-received. I also think that we need to think about what the user agent can do for the user in light of the location of things on the Web, e.g. geofencing.

**Comment by Malvoz** [comment]
Concerning the issue with overflowing parent scrolling along when using arrow keys, I [opened an issue](https://github.com/Leaflet/Leaflet/issues/6229) and it was indeed considered a bug with Leaflet.

**Comment by prushforth** [comment]
OK thanks for looking into this. I will have a look as well.

**Comment by prushforth** [comment]
@malvoz Thanks to the Leaflet team's @cherniavskii with a bug fix so quickly in response to your issue, I was able to patch the copy of Leaflet we're using and draft a patch release.  

So when someone installs or updates (edit: maybe not, I will have to look into how to make this happen better) their copy of web-map component with bower they will get the fixed version.  I also deployed it to the [dev site](http://geogratis.gc.ca/api/beta/mapml/en/cbmtile/cbmt/#12,-90.6855534,50.6741905). If you've been there in the past week you will have to refresh your cache hard.

Thanks for taking the time to notice this.

**Comment by Malvoz** [comment]
@prushforth The link to the dev site in your last comment points to localhost and I can't figure out where it's supposed to take me, please update it!

**Comment by prushforth** [comment]
The dev site is http://geogratis.gc.ca/api/beta/mapml/

**Comment by cherniavskii** [comment]
@prushforth you are welcome :)
I'll try to find some time to release next version of Leaflet this week.

**Comment by Malvoz** [comment]
This issue was mostly overtaken by a11y issues in the polyfill. I don't think there is any action to be taken here. We have a separate issue for the MapML Accessibility API Mappings.
